os/drivers/pm/pm.c

```diff
@@ -96,11 +97,11 @@ static int pm_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 	/* Handle built-in ioctl commands */
 	switch (cmd) {
     	case PMIOC_SUSPEND:
-	        ret = pm_suspend((enum pm_domain_e)arg);
+	        ret = pm_suspend((int)arg);
 		pmvdbg("State locked!\n");
```

Comment:
Don't we check to return value?

Reply:
yes pm_suspend returns OK or ERROR, and if error is setting the errno